tests/fixtures/yaml_tests/failing_test_variable_absolute_error_margin.yaml

```diff
@@ -0,0 +1,9 @@
+- name: "Failing test: result out of variable specific absolute error margin"
+  period: 2015-01
+  absolute_error_margin:
+    default: 100
+    income_tax: 50
```

Comment:
Ah, so here you declare an absolute for income tax overriding the abdolute of the calculaiton ?